src/relations/db.py

```diff
@@ -291,11 +291,11 @@ def _get_state(self) -> str:
         """Gets the given state for this unit.
 
         Returns:
-            The state of this unit. Can be 'standalone', 'master', or 'standby'.
+            The state of this unit. Can be 'standalone', 'primary', or 'standby'.
         """
         if len(self.charm._peers.units) == 0:
             return "standalone"
         if self.charm._patroni.get_primary(unit_name_pattern=True) == self.charm.unit.name:
-            return "master"
+            return "primary"
```

Comment:
This probably will break legacy relations, as charms expect this value in this specific field from the relation.